packages/lit-html/src/directive-helpers.ts

```diff
@@ -149,31 +149,33 @@ export const setPartValue = <T extends Part>(
 const RESET_VALUE = {};
 
 /**
- * Resets the stored state of a ChildPart used for dirty-checking and
- * efficiently updating the part to the given value, or when omitted, to a state
- * where any subsequent dirty-check is guaranteed to fail, causing the next
- * commit to take effect.
+ * Resets the committed value a ChildPart directly without triggering the
+ * commit stage of the part.
+ *
+ * This is useful in cases where a directive needs to update the part such
+ * that the next update detects a value change or not.
```

Comment:
Add back the note describing the effect of omitting `value`? 

Something like:
> When `value` is omitted, the next update will be guaranteed to be detected as a change.

Reply:
done